### ISSUE TITLE
Build and Use This Game

### ISSUE THREAD

**Issue by CryptoDevving** [saha0073/Multiplayer-Blockchain-Game-with-Cryptocurrency-Transaction#1]
Am I allowed to build on and use this game with my cryptocurrency?

**Comment by saha0073** [comment]
Yeah, you can build on this with your cryptocurrency. To build this game, I
used the Unity demo sample provided by Moralis Metaverse SDK.
